src/activity_item_handling.cpp

```diff
@@ -875,6 +875,44 @@ static int move_cost( const item &it, const tripoint &src, const tripoint &dest
     return move_cost_inv( it, src, dest );
 }
 
+static void vehicle_repair_activity( player &p, const tripoint src_loc, int vpindex )
+{
+    vehicle *veh = veh_pointer_or_null( g->m.veh_at( src_loc ) );
+    if( !veh ) {
+        return;
+    } else {
```

Comment:
```suggestion
    }
```
And now you can remove one level of indentation from the remaining code in this function.